e2e/raw-objects/samples/rollout-demo/README.md

```diff
@@ -11,7 +11,7 @@ helm install -n flagger rollout charts/flagger/
 ## Install Vela core
 ```shell script
 make
-bin/vela install
+helm install -n vela-system kubevela charts/vela-core 
```

Comment:
please leave it as it is. Or complete the whole instruction:

1. add helm repo
2. update repo

Reply:
I agree